js/locations.js

```diff
@@ -1,4 +1,4 @@
-var locations = [
+mvar locations = [
```

Comment:
Accidental typo here?

Reply:
Yes. I should not do this on my smartphone.